### PR TITLE
Initial setup

### PR DESCRIPTION
**Pull request by nowyDEV**
I think we should get rid of "recommended" rules that come from plugins and list them (and their values) explicitly.
This will require a lot of work but It's more safe i think (plugins may change they recommended rules after update).

### CODE REVIEW

**Review by mrMarshalX — APPROVED** [review]

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# What
+
+- What was done in this Pull Request
+- How was it before
+- Any other information that may be useful for the dev to review Pull Request
+
+## Compatibility
+
+- [ ] Does this change maintain backward compatibility?
```
Comment (Bosmanfrx) [inline .github/PULL_REQUEST_TEMPLATE.md:9]: Does any linter change guarantee backward compatibility?

Comment (nowyDEV) [reply .github/PULL_REQUEST_TEMPLATE.md:9]: Yep, e.g. update of dependencies, extending docs, adding additional features (like other config for another tool)

File: .github/workflows/release.yaml [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+        with:
+          node-version: "16"
+      - run: echo "registry=https://registry.npmjs.org/" > .npmrc
+      - run: echo "//registry.npmjs.org/:_authToken=${NODE_AUTH_TOKEN}" >> .npmrc
+        env:
+          NODE_AUTH_TOKEN: ${{ secrets.NPM_KEY }}
+      - run: cat .npmrc
```
Comment (crabtree) [inline .github/workflows/release.yaml:17]: Is it required?

File: README.md [hunk 3]
```diff
@@ -1,2 +1,146 @@
...
+To install the package, do this:
+
+```sh
+cd your-project-folder
+npm install -D eslint typescript prettier @nordcloud/eslint-config-pat
+```
+
+### 2. Choose one profile
+
+The ruleset currently supports three different "profile" strings, which select lint rules applicable for
```
Comment (crabtree) [inline README.md:36]: You are mentioning three different profiles but below there are only two enumerated.

File: .github/workflows/release.yaml (hunk 2 above)
Comment (nowyDEV) [reply .github/workflows/release.yaml]: Nope

**Review by crabtree — APPROVED** [review]

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (Bosmanfrx) [reply .github/PULL_REQUEST_TEMPLATE.md:9]: Ah, that's great. Thank you!

File: prettier.config.js [hunk 4]
```diff
@@ -0,0 +1,27 @@
+module.exports = {
+  arrowParens: "always",
+  bracketSpacing: true,
+  embeddedLanguageFormatting: "auto",
+  htmlWhitespaceSensitivity: "css",
+  insertPragma: false,
+  bracketSameLine: false,
+  jsxSingleQuote: false,
+  printWidth: 80,
```
Comment (Bosmanfrx) [inline prettier.config.js:9]: :'(

**Review by Bosmanfrx — APPROVED** [review]

**Review by mrMarshalX — APPROVED** [review]

File: prettier.config.js (hunk 4 above)
Comment (nowyDEV) [reply prettier.config.js:9]: You can always overwrite it per-project ;d

**Pull request merged**